src/cargo/core/compiler/mod.rs

```diff
@@ -711,6 +714,11 @@ fn prepare_rustc(build_runner: &BuildRunner<'_, '_>, unit: &Unit) -> CargoResult
 
     if is_primary {
         base.env("CARGO_PRIMARY_PACKAGE", "1");
+
+        if gctx.cli_unstable().sbom && build_runner.bcx.build_config.sbom {
+            let file_list = std::env::join_paths(build_runner.sbom_output_files(unit)?)?;
+            base.env("CARGO_SBOM_PATH", file_list);
```

Comment:
Can you document this in the unstable docs like it will be when moved to the stable docs?

I want to make sure we call out that it can be multiple files